test/t8n/t8n.cpp

```diff
@@ -80,6 +83,7 @@ int main(int argc, const char* argv[])
         j_result["currentDifficulty"] = "0x20000";
         j_result["currentBaseFee"] = hex0x(block.base_fee);
 
+        int64_t cumulativeGasUsed = 0;
```

Comment:
```suggestion
        int64_t cumulative_gas_used = 0;
```

Reply:
Shit. Sorry for this. Too fast as always.